index.md

```diff
@@ -8,29 +8,29 @@ active_tab: main_page
 ---
 
 
-Course number
+Course number:
```

Comment:
I think I squashed the commits.

On Thu, Aug 13, 2015 at 12:46 PM, JACK GINDI <jgindi@sas.upenn.edu> wrote:

> No it doesn't.
>
> On Thu, Aug 13, 2015 at 12:09 PM, Jim Chen <notifications@github.com>
> wrote:
>
>> In index.md
>> <https://github.com/callison-burch/data-structures-and-algorithms/pull/1#discussion_r37013360>
>> :
>>
>> > @@ -8,29 +8,29 @@ active_tab: main_page
>> >  ---
>> >
>> >
>> > -Course number
>> > +Course number:
>>
>> Doesn't this cause two :s to be added? There is one at the beginning of
>> the next line.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/callison-burch/data-structures-and-algorithms/pull/1/files#r37013360>
>> .
>>
>
>